### DISCUSSION TITLE
How does map work? How does zigbee mesh work?

### DISCUSSION THREAD

**Discussion by evlo** [Koenkk/zigbee2mqtt#15141] [General]
I have created network for only tuya devices (+ 1 awox / eglo light)

The awox thing is suepr hard to pair, so I pared it using permit join all.

My question:
- why do devices in red rectangles, mains powerered, connect trough stuff that is not close to them (stuff in purple rectangles are ideal nodes)?
*i think i even paired the "ponk" socket trougth the "meter", but it went for the coordinator that it does not have sight to), this is same for months  ... well acutally ideal connection would be "ponk socket" to awox light and then awox light to one of the "meters"*
- how can stuff in yellow rectangles be connected to the network and working if coordinator does not see any path from it to the devices?
- is there a way to keep device paired but force it to find new best route to coordinator?

![image](https://user-images.githubusercontent.com/8813196/203532386-b447af3c-5d6d-4c38-b36c-8b4e1e5986c1.png)

**Comment by wwebers** [comment]
Same here. Am also experiencing several "isolated" devices which still work fine. Wouldn't give too much attention on that "map". Seem not to reflect the reality.
